code/modules/reagents/reagent_containers.dm

```diff
@@ -61,10 +61,10 @@
 /obj/item/reagent_containers/afterattack(obj/target, mob/user , flag)
 	return
 
-/obj/item/reagent_containers/proc/reagentlist(obj/item/reagent_containers/snack) //Attack logs for regents in pills
+/obj/item/reagent_containers/proc/reagentlist() //Return reagents in a reagent_container, default to source

```

Comment:
See now why the way it was before was so utterly stupid?

Reply:
I practically cringed when I saw the default argument specify it must be a snack..

Reply:
I could try to move it to atom if you want